guides/working-with-examples.md

```diff
@@ -0,0 +1,47 @@
+# Working with Examples
+
+Adding meaningful examples can greatly enrich the documentation of components in the Cashmere library. Here are some things to remember when creating or modifying examples.
+
+## Examples files
+
+The following files are used in the creation of examples for the Cashmere documentation website.
+
+### Files you may need to modify
+
+-   `projects/cashmere-examples/src/lib/{EXAMPLE_NAME}` - the folder containing all files for an example
+-   `src/app/core/document-items.service.ts` - the file that adds the example to the appropriate component page in the docs site
+
+### Files used in the build process (which you shouldn't need to modify)
+
+-   `src/assets/docs/examples` - the output directory for StackBlitz example project files
+-   `projects/cashmere-examples/src/lib/cashmere.module.ts` - if your example contains a module file, you will need to import this CashmereModule
+-   `projects/cashmere-examples/src/lib/cashmere.generated.module.ts` - a generated copy of `src/app/shared/cashmere.module.ts`, used for both examples and StackBlitz
+-   `projects/cashmere-examples/src/lib/example-mappings.generated.ts` - this generated file maps example names to the main component for the example, used by the `ExampleViewerComponent`
+-   `projects/cashmere-examples/src/lib/examples.generated.module.ts` - the generated module file containing all the examples, imported into the documentation site for rendering the examples
+-   `projects/cashmere-examples/src/project-template` - the folder containing Angular CLI project files used in the StackBlitz
+-   `scripts/build-examples.ts` - the script used to generate the support files needed for the docs site and StackBlitz
+
+## Creating a new example
+
+_Please pay close attention to the naming conventions used here, otherwise your example may not work on the documentation site and/or StackBlitz._
+
+1. Name the example (the [hardest part](https://bit.ly/2VI8ZI5)). Figure out what you want the title of the example to be on the documentation site.
+2. Derive the example name from the example title by converting it to kebab case (for instance, if your example is titled "Format Hard Drive", the example name would be `format-hard-drive`)
+3. Create a folder under `projects/cashmere-examples/src/lib/` named your example name, i.e. `projects/cashmere-examples/src/lib/format-hard-drive`
```

Comment:
nitpicky, but its usually called directory, not folder

Reply:
noted